webhooks-extension/pkg/endpoints/sink.go

```diff
@@ -227,6 +230,99 @@ func createPipelineRunFromWebhookData(buildInformation BuildInformation, r Resou
 	logging.Log.Debugf("PipelineRun created: %+v.", pipelineRun)
 }
 
+// This is the task creation for the minitoring the main pipelinerun and reporting the resule to the github
+func createTaskRunFromWebhookData(buildInformation BuildInformation, r Resource) {
+	logging.Log.Debugf("In createTaskRunFromWebhookData, build information: %s.", buildInformation)
+
+	// Install namespace
```

Comment:
I know it's copied from elsewhere but let's remove this comment as it's pretty obvious

Reply:
done